crates/aptos/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "aptos"
 description = "Aptos tool for management of nodes and interacting with the blockchain"
-version = "4.2.5"
+version = "4.2.6"
```

Comment:
I think `Cargo.lock` also needs to be updated (similar to here: https://github.com/aptos-labs/aptos-core/pull/13187/files).

Reply:
Yeah, just a `cargo build`, you can cancel almost immediately, but it will update the `Cargo.lock`